### PR TITLE
Dnssec05

### PR DESCRIPTION
**Pull request by mattias-p**
Implementation of zonemaster/zonemaster#626.

### CODE REVIEW

File: lib/Zonemaster/Engine/Constants.pm [hunk 1]
```diff
@@ -55,11 +57,13 @@ our %EXPORT_TAGS = (
...
-Readonly our $ALGO_STATUS_DEPRECATED => 1;
-Readonly our $ALGO_STATUS_PRIVATE    => 4;
-Readonly our $ALGO_STATUS_RESERVED   => 2;
-Readonly our $ALGO_STATUS_UNASSIGNED => 3;
-Readonly our $ALGO_STATUS_VALID      => 5;
+Readonly our $ALGO_STATUS_DEPRECATED   => 1;
+Readonly our $ALGO_STATUS_PRIVATE      => 4;
+Readonly our $ALGO_STATUS_RESERVED     => 2;
+Readonly our $ALGO_STATUS_UNASSIGNED   => 3;
+Readonly our $ALGO_STATUS_VALID        => 5;
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Constants.pm]: What does "valid" mean?

File: lib/Zonemaster/Engine/Test/DNSSEC.pm [hunk 2]
```diff
@@ -22,23 +22,28 @@ use Carp;
...
+        sig         => 0,
     },
     1 => {
         status      => $ALGO_STATUS_DEPRECATED,
         description => q{RSA/MD5},
         mnemonic    => q{RSAMD5},
+        sig         => 0,
     },
     2 => {
         status      => $ALGO_STATUS_VALID,
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Test/DNSSEC.pm]: What is this "valid" for?

File: lib/Zonemaster/Engine/Constants.pm (hunk 1 above)
Comment (mattias-p) [reply lib/Zonemaster/Engine/Constants.pm]: I interpret it as "non of the others". If you have a better name I can change it.

File: lib/Zonemaster/Engine/Test/DNSSEC.pm (hunk 2 above)
Comment (mattias-p) [reply lib/Zonemaster/Engine/Test/DNSSEC.pm]: This is the same comment as above. See above for response.

File: lib/Zonemaster/Engine/Constants.pm (hunk 1 above)
Comment (matsduf) [reply lib/Zonemaster/Engine/Constants.pm]: $ALGO_STATUS_OTHER

**Pull request merged**